types/jsforce/jsforce-tests.ts

```diff
@@ -30,9 +41,9 @@ salesforceConnection.sobject("ContentVersion").create({
     }
 });
 
-salesforceConnection.sobject("ContentVersion").retrieve("world", {
+salesforceConnection.sobject<any>("ContentVersion").retrieve("world", {
```

Comment:
This is a slight change, but is intended to be used with module augmentation as shown in the comment to allow for seamless type inference on a per-collection basis.